README.md

```diff
@@ -1 +1,5 @@
-# bibliopompica
\ No newline at end of file
+# bibliopompica
+
+Build assets: `npm run build`
+
+Run test server: `npm run server`
```

Comment:
So we don't need to interact with Flask/Python?

Reply:
OH I see in the json... guess I should've kept reading before commenting :P

Reply:
I kept forgetting the environment variable for flask :p